numpy/core/tests/test_longdouble.py

```diff
@@ -71,6 +71,39 @@ def test_fromstring():
                  err_msg="reading '%s'" % s)
 
 
+def test_fromstring_complex():
+    for ctype in ["complex", "cdouble", "cfloat"]:
+        # Check spacing between separator
+        assert_equal(np.fromstring("1, 2 ,  3  ,4",sep=",",dtype=ctype),
```

Comment:
Small nits also here: Please make sure to put spaces after all commas in function arguments as per PEP8.